### DISCUSSION TITLE
Map folder location

### DISCUSSION THREAD

**Discussion by babylard** [redeclipse/base#1392] [Help and Support] [answered]
Hello!

I was trying to find the folder where maps are stored for Red Eclipse 2.
I am running macOS Monterey. does anyone know where the folder is? 

Thank you!

**Comment by babylard** [comment]
Oh and also I downloaded it through Steam if that is helpful.

**Comment by dc-redeclipse** [comment, marked as answer]
Hey welcome!
under MacOS (noneSteam) check the app data in your home directory: /Users/<username>/Library/Application Support/Red Eclipse

under Steam its kinda caped away a bit under your local Steamfiles

btw. Red Eclipse is for now on just Red Eclipse as it was bevor the Red Eclipse 2 was only on steam for a short time. Its just Red Eclipse there to and for the next release to sort out the naming situation.
Check out thr [FAQ](https://github.com/tempris/redeclipse-docs/blob/master/FAQ.md) if you need adition Help on this or join us on [discord](https://www.redeclipse.net/discord).

**Reply by babylard** [reply]
Thank you!

**Reply by qreeves** [reply]
> under Steam its kinda caped away a bit under your local Steamfiles

Nah, it's in the same location, Steam will just also upload some of the files to the Steam Cloud backup.
